lib/dl_connector_trino/dl_connector_trino_tests/db/api/test_data.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+
+import pytest
+
+from dl_api_client.dsmaker.api.data_api import SyncHttpDataApiV2
+from dl_api_client.dsmaker.api.dataset_api import SyncHttpDatasetApiV1
+from dl_api_client.dsmaker.primitives import Dataset
+from dl_api_client.dsmaker.shortcuts.result_data import get_data_rows
+from dl_api_lib_testing.connector.data_api_suites import (
+    DefaultConnectorDataCacheTestSuite,
+    DefaultConnectorDataDistinctTestSuite,
+    DefaultConnectorDataGroupByFormulaTestSuite,
+    DefaultConnectorDataPreviewTestSuite,
+    DefaultConnectorDataRangeTestSuite,
+    DefaultConnectorDataResultTestSuite,
+)
+from dl_api_lib_testing.data_api_base import DataApiTestParams
+from dl_constants.enums import WhereClauseOperation
+from dl_core_testing.database import Db
+
+from dl_connector_trino_tests.db.api.base import TrinoDataApiTestBase
+
+
+class TestTrinoDataResult(TrinoDataApiTestBase, DefaultConnectorDataResultTestSuite):
+    def test_datetime_filter_with_zulu_timezone(
```

Comment:
Isn't this test just a copy-n-paste from base except for the `control_api` fixture? Or is this fixture the actual reason to override it